src/main/java/com/pi4j/crowpi/components/UltrasonicDistanceSensorComponent.java

```diff
@@ -76,12 +150,85 @@ public double measure(double temperature) {
         return calculateDistance(pulseLength, temperature);
     }
 
+    /**
+     * Sets or disables the handler for the object found recognition.
+     * This event gets triggered whenever a object comes into the specified range
+     * Only a single event handler can be registered at once.
+     *
+     * @param min Minimum distance to the Object in cm
+     * @param max Maximum distance to the object in cm
+     * @param handler Event handler to call or null to disable
```

Comment:
Can you please indent these Javadoc params correctly, e.g. so that all comments/descriptions are at the same position? Auto-formatting should fix this.